usr/lib/cca_stdll/cca_specific.c

```diff
@@ -8500,6 +8502,356 @@ static CK_RV cca_mk_change_cancel_query(STDLL_TokData_t *tokdata,
     return rc;
 }
 
+static CK_RV cca_reencipher_sec_key(STDLL_TokData_t *tokdata,
+                                    struct cca_mk_change_op *mk_change_op,
+                                    CK_BYTE *sec_key, CK_BYTE *reenc_sec_key,
+                                    CK_ULONG sec_key_len, CK_BBOOL from_old)
+{
+    enum cca_token_type keytype;
+    unsigned int keybitsize;
+    const CK_BYTE *mkvp = NULL;
+    const CK_BYTE *new_mkvp = NULL;
+    const char *mk_type;
+    const char *verb;
+    enum cca_ktc_type ktc_type;
+    unsigned char rule_array[256] = { 0, };
```

Comment:
also here you may use constant CCA_RULE_ARRAY_SIZE

Reply:
fixed